ex04.ex

```diff
@@ -75,10 +82,14 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
-
-
+  def even_odd_check(head, {even, odd}) when is_number(head) and Integer.is_even(head) do
```

Comment:
In general you want to avoid concatenating lists unless you know they are going to be short.